src/ontology/edits/oeo-physical.omn

```diff
@@ -1659,10 +1659,12 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/350",
 Class: OEO_00000207
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "In thermodynamics, heat is energy in transfer to or from a thermodynamic system, by mechanisms other than thermodynamic work or transfer of matter.
-       The mechanisms include conduction, through direct contact of immobile bodies, or through a wall or barrier that is impermeable to matter; or radiation between separated bodies; or isochoric mechanical work done by the surroundings on the system of interest; or Joule heating by an electric current driven through the system of interest by an external system; or a combination of these."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Heat&oldid=90563031157"@en,
-        rdfs:label "heat"
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Thermal energy is the energy that a material entity contains in the undirected motion of its constituent parts (e.g. molecules and atoms)."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000116> "\"Heat\", although it strictly speaking describes a mode of energy transfer, not an energy, is included as a colloquialism for \"thermal energy\", following everyday usage."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "heat"@en,
```

Comment:
There was no consensus in #522 to add the alternative term `heat` without adding a second class for the thermal energy transfer. So for the moment the alternative term should be excluded, else we would get a contradiction in the OEO:  If an alternative term is solely applied to one class that means it is a synonym. At the same time the explanatory note states that it is not a synonym.

Reply:
There is another annotation type, called "has related synonym", this might fit better than "alternative term".

Reply:
I think adding it with the "has related synonym" annotation is a good compromise.

Reply:
And what is a "related synonym" supposed to be?  
http://purl.obolibrary.org/obo/IAO_0000118 is quite explicit
> An alternative name for a class or property which means the same thing as the preferred name (semantically equivalent) 

Which applies here in so far as it is an colloquialism.  People say "heat" when they mean "thermal energy", although heat is technically different from thermal energy.  But language is fuzzy like that.

http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym on the other hand leads to some xml dump.

Digging around, on finds that `hasRelatedSynonym` is &ast;drum roll&ast; … [undefined](http://www.ontobee.org/ontology/GO?iri=http://www.geneontology.org/formats/oboInOwl%23hasRelatedSynonym) (as are "has_broad_synonym", "has_exact_synonym", and  "has_narrow_synonym").

So if the alternative to using a well-defined annotation we can't agree upon is to select seemingly at random on of four undefined annotations that kinda sound like they could have something to do with it, I'll opt for not annotating at all.

Reply:
> http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym on the other hand leads to some xml dump.

This document actually contains definitions for `has related synonym` and the other synonym annotations (although they are indeed not easy to find).

`has related synonym` is defined as _An alias in which the alias is related the primary class name, but not necessarily broader or narrower. Example: cytochrome bc1 complex is a related synonym of ubiquinol-cytochrome-c reductase activity; virulence is a related synonym of pathogenesis_

I'm not sure if this fits better than `alternative term`. It might be best to do it like @l-emele suggested and include the alternative term "heat" when we also have a class "heat".

Reply:
I'm fine with waiting for `heat`.